app/controllers/subscription_controller.rb

```diff
@@ -109,6 +109,13 @@ def delete
       end
     end
   end
+  
+  def notes
+    User.content_followed_in_past_period(time_period)
```

Comment:
So here, would you do this /only/ for the current user? 

So then, `current_user.content_followed...`

To very specifically define what you're trying to do here, maybe a functional test is in order? And a unit test for `content_followed...` could help here too. But you're off to a great start!!!